### PR TITLE
Release/v0.1.0

### PR DESCRIPTION
**Pull request by mwnicks**
Created first release of the new widget server.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+// var express = require('express');
```
Comment (nshumoogum) [inline server.js]: Commented out code?

**Review by nshumoogum — CHANGES_REQUESTED** [review]

**Review by nshumoogum — COMMENTED** [review]
Would be nice to add a _README.md_, _LICENSE.md_, _CONTRIBUTING.md_ and _.github/PULL_REQUEST_TEMPLATE.MD_, the last three can be copied from other repositories

File: server.js (hunk 1 above)
Comment (mwnicks) [reply server.js]: I'd forgotten that was there.
Removed now

**Comment by mwnicks** [comment]
> PULL_REQUEST_TEMPLATE.MD

Documentation added

**Review by nshumoogum — APPROVED** [review]

**Pull request merged**